files/config/default/etc/org.opencastproject.ui.metadata.CatalogUIAdapterFactory-series-uct.cfg

```diff
@@ -67,6 +67,15 @@ property.caption-type.required=false
 property.caption-type.namespace=http://media.uct.ac.za/uct/video
 property.caption-type.order=6
 
+# List of AI-generated transcript features shown to users for this series
+property.transcription-list.inputID=transcript-features
+property.transcription-list.label=AI Features
+property.transcription-list.type=mixed_text
```

Comment:
> For mixed_text and iterable_text type fields, a string at which inputs into the corresponding fields are split into individual values for easier bulk entry of lists. The default is no delimiter, in which case no splitting takes place.

https://docs.opencast.org/r/16.x/admin/#configuration/metadata/#part-3-catalog-fields-configuration

Did you want to make it mixed_text field? without splitting? Otherwise please then add a delimiter definition
`property.transcription-list.delimiter=;`

Reply:
Thanks Corne. Done!